libs/designer-ui/src/lib/picker/picker.tsx

```diff
@@ -24,16 +24,32 @@ export interface PickerProps {
   currentPathSegments: IBreadcrumbItem[];
   files: FileItem[];
   loadingFiles?: boolean;
+  errorDetails?: { message: string };
   onCancel?: () => void;
   handleFolderNavigation?: (selectedObject: any) => void;
+  handleFolderSelected?: (selectedObject: any) => void;
```

Comment:
It is better to call this as handleItemSelected to make it generic for file/folder